### ISSUE TITLE
Many packages in gamma.3 release miss ESM distributions

### ISSUE THREAD

**Issue by AllanFly120** [aws/aws-sdk-js-v3#1364]
**Describe the bug**
With #1297 ideally all the packages should be released with ESModule and CJS module. But appearantly not all the packages contains ESM. 
For example most of the packages under `packages` constains `./dist/cjs` exists, `./dist/es` is missing.

**SDK version number**
Example: @aws-sdk/middleware-serde@1.0.0-gamma.2

**Is the issue in the browser/Node.js/ReactNative?**
Browser

**To Reproduce (observed behavior)**
`yarn add @aws-sdk/middleware-serde@1.0.0-gamma.2` and find the `node_modules/@aws-sdk/middleware-serde/dist/es`

**Expected behavior**
`node_modules/@aws-sdk/middleware-serde/dist/es` should exist.

**Screenshots**
<details>
<summary> click to unfold </summary>

![Screen Shot 2020-07-10 at 4 55 12 PM](https://user-images.githubusercontent.com/7614947/87211399-2a7b5a00-c2ce-11ea-9213-3336f97b3dc2.png)

</details>

**Comment by trivikr** [comment]
Checked with `@aws-sdk/middleware-serde@1.0.0-gamma.3`, `node_modules/@aws-sdk/middleware-serde/dist/es` still doesn't exist

**Comment by trivikr** [comment]
Verified that @aws-sdk/middleware-serde is released as es from `1.0.0-gamma.4` onwards.

To verify:
```console
$ yarn add @aws-sdk/middleware-serde@gamma

$ ls node_modules/@aws-sdk/middleware-serde/dist/
cjs	es
```
